src/main/resources/templates/fragments/footer.html

```diff
@@ -13,22 +13,68 @@
         </footer>
         <!-- End Footer -->
 
-        <script src="https://code.jquery.com/jquery-2.1.3.min.js"></script>
         <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
                 integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
                 crossorigin="anonymous"></script>
         <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
                 integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
                 crossorigin="anonymous"></script>
-        <script type="text/javascript">
-            function ConfirmDelete() {
-                return confirm("Are you sure you want to delete?");
+<!--        For Delete Item-->
+        <script language="javascript" type="text/javascript">
+            function ConfirmDelete(id) {
+                if (window.confirm('Are you sure you want to delete?')) {
+                    $.ajax({
+                        type : "DELETE",
+                        url : "/delete/"+ id,
+                        success : function() {
+                            console.log("a mers");
+                            window.location.reload();
+                        },
+                        error : function() {
+                            alert("Something didn't work well. If the error persist, please contact us!");
+                        }
+                    });
+                }
             }
         </script>
-        <script type="text/javascript">
-            function ConfirmDeleteCategory() {
-                return confirm("Are you sure you want to delete?" +
-                 "\nWhen you delete this category, all the items who belong of this category, will be delete!");
+<!--        For Delete Item from View page-->
+        <script language="javascript" type="text/javascript">
+            function ConfirmDeleteFromView(id) {
```

Comment:
is the same function as ConfirmDelete, let's don't have repetitive

Reply:
It is not completely repeated. In the end, one reloads to the current page, and the other returns you to the previous page. It depends on where the deletion is done.

Reply:
I was thinking of having a single function that I could split in two and distinguish the differences by an "if" in which to do something if it comes from page x and to do something else if it comes from page y. But I gave up, because, anyway, in the next future, I won't need the "... FromView" function anymore, so it won't be so crowded anymore.